tags/update.md

```diff
@@ -4,8 +4,8 @@ color: blue
 aliases: ['updating', 'autoupdate']
 ---
 
-Windows auto-updating is WIP. For now, you will need to download the installer and run it again in order to update. You will not lose your instances.
+Windows auto-updating is only supported on 8.0+. For now, you will need to download the installer and run it again in order to update. You will not lose your instances.
```

Comment:
```suggestion
Windows auto-updating is only supported on 8.0+. For now, you will need to download the installer and run it again in order to update. You will not lose your instances.
```

For now? It's the past now

Maybe you could say "if you are still on a previous version"?

Reply:
Oh yeah I'll make some changes.

Reply:
How about now?